src/main/resources/com/conveyal/gtfs/storage/sql_statements.sql

```diff
@@ -0,0 +1,32 @@
+-- create_feeds_table
```

Comment:
What is the purpose of this file?

Reply:
I was originally loading pre-written SQL statements from a file before I started generating them programmatically. I'm deleting this file.